aspnetcore/tutorials/razor-pages/model.md

```diff
@@ -1,44 +1,472 @@
 ---
 title: Part 2, add a model to a Razor Pages app in ASP.NET Core
 author: rick-anderson
-description: Part 2 of tutorial series on Razor Pages.
+description: Part 2 of tutorial series on Razor Pages. In this section model classes are added.
 ms.author: riande
-ms.date: 12/05/2019
+ms.date: 09/30/2020
 no-loc: ["ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
 uid: tutorials/razor-pages/model
 ---
 # Part 2, add a model to a Razor Pages app in ASP.NET Core
 
 By [Rick Anderson](https://twitter.com/RickAndMSFT)
 
-::: moniker range=">= aspnetcore-3.0"
+::: moniker range=">= aspnetcore-5.0"
+
+<!-- In the next update on the CLI version, let the scaffolder do the same work the VS driven scaffolder does. That is, create the DB context, etc -->
+
+In this section, classes are added for managing movies in a database. The app's model classes use [Entity Framework Core (EF Core)](/ef/core) to work with the database. EF Core is an object-relational mapper (O/RM) that simplifies data access. You write the model classes first, and EF Core creates the database.
+
+The model classes are known as POCO classes (from "**P**lain-**O**ld **C**LR **O**bjects") because they don't have any dependency on EF Core. They define the properties of the data that are stored in the database.
+
+[View or download sample code](https://github.com/dotnet/AspNetCore.Docs/tree/master/aspnetcore/tutorials/razor-pages/razor-pages-start/sample/RazorPagesMovie50) ([how to download](xref:index#how-to-download-a-sample)).
+
+## Add a data model
+
+# [Visual Studio](#tab/visual-studio)
+
+1. Right-click the **RazorPagesMovie** project > **Add** > **New Folder**. Name the folder *Models*.
+1. Right-click the *Models* folder. Select **Add** > **Class**. Name the class **Movie**.
+1. Add the following properties to the `Movie` class:
+
+[!code-csharp[](~/tutorials/razor-pages/razor-pages-start/sample/RazorPagesMovie22/Models/Movie.cs?name=snippet1)]
+
+The `Movie` class contains:
+
+* The `ID` field is required by the database for the primary key.
+* `[DataType(DataType.Date)]`:  The [DataType](xref:System.ComponentModel.DataAnnotations.DataTypeAttribute) attribute specifies the type of the data (Date). With this attribute:
+
+  * The user is not required to enter time information in the date field.
```

Comment:
Prefer contractions when you can use them. Acrolinx scores are often higher when using them.

```suggestion
  * The user isn't required to enter time information in the date field.
```